lmdeploy/pytorch/nn/norm.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Any
+
+import torch
+from torch import nn
+
+from ..backends import LayerType, get_backend
+
+
+def _is_w8a8(quant_config: Any):
+    """is w8a8."""
+    if quant_config is None:
+        return False
+    else:
+        quant_method = quant_config['quant_method']
+        if quant_method == 'w8a8':
```

Comment:
I recommend "smooth_quant" instead of `w8a8`

Reply:
Follow https://github.com/InternLM/lmdeploy/pull/2373